lighthouse-core/report/report-generator.js

```diff
@@ -66,19 +66,23 @@ class ReportGenerator {
     const separator = ',';
     /** @param {string} value @return {string} */
     const escape = value => `"${value.replace(/"/g, '""')}"`;
+    /** @param {Array<string | number>} row @return {string[]} */
+    const rowFormatter = row => row.map(value => value.toString()).map(escape);
 
     // Possible TODO: tightly couple headers and row values
     const header = ['requestedUrl', 'finalUrl', 'category', 'name', 'title', 'type', 'score'];
     const table = Object.values(lhr.categories).map(category => {
-      return category.auditRefs.map(auditRef => {
+      const rows = [];
+      const overallCategoryScore = category.score === null ? -1 : category.score;
+      rows.push(rowFormatter([lhr.requestedUrl, lhr.finalUrl, category.title,
+        'overall-category-score', 'Overall Category Score', 'numeric', overallCategoryScore]));
```

Comment:
WDYT about including the category in these column values? something like the below

```suggestion
    const table = Object.keys(lhr.categories).map(categoryId => {
      const rows = [];
      const category = lhr.categories[categoryId];
      const overallCategoryScore = category.score === null ? -1 : category.score;
      rows.push(rowFormatter([lhr.requestedUrl, lhr.finalUrl, category.title,
        '${categoryId}-score', `${category.title} Category Score`, 'numeric', overallCategoryScore]));
```

Reply:
Yeah it would be more readable. I will make these changes.

Reply:
Do you think we need to iterate over keys also along with values? Can we use `category.title` to get the category name to include in the column values

Reply:
Oh, no we definitely don't need to iterate both. GitHub wouldn't allow a multi-line suggestion so this would absolutely *replace* the `Object.values` iteration (we pull in the value with `const category = lhr.categories[categoryId];` here anyway).

Does that make sense?

Reply:
Yeah it makes sense. I modified the code. Please review